docs/source/pythonapi/index.rst

```diff
@@ -29,7 +29,7 @@ Classes
    :template: myclass.rst
 
    openmc.XSdata
-   openmc.MGXSLibraryFile
+   openmc.MGXSLibrary
```

Comment:
As an unrelated comment to this PR, but a thought that I've had for some time - we may want to include in the documentation a note that the `openmc.MGXSLibrary` class is fundamentally separate from the `openmc.mgxs` module. At the moment our API for MGXS may appear a bit scattered to a new user, and it would be nice to provide a little direction to point out the different use cases for each. This is esp. true since we don't yet have any thorough examples of MG mode. What do you think @paulromano?